pkg/kubelet/kubelet.go

```diff
@@ -2055,3 +2062,20 @@ func ParseReservation(kubeReserved, systemReserved utilconfig.ConfigurationMap)
 	}
 	return reservation, nil
 }
+
+// Gets the streaming server configuration to use with in-process CRI shims.
+func getStreamingConfig(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *KubeletDeps) *streaming.Config {
+	config := &streaming.Config{
+		// Use a relative redirect (no scheme or host).
+		BaseURL: &url.URL{
+			Path: "/cri/",
```

Comment:
Yes. The relative URL is resolved here: https://github.com/kubernetes/kubernetes/pull/34987/files#diff-4b9b9659e8f0abce3dab96b924f6b107R312
I also verified this works manually.